In [1]:
import pandas as pd

In [2]:
data = pd.read_csv('FoodPrice_in_Turkey.csv')

In [3]:
data.head()

Place  ProductId    ProductName  UmId UmName  Month  Year  \
0  National Average         52  Rice - Retail     5     KG      5  2013   
1  National Average         52  Rice - Retail     5     KG      6  2013   
2  National Average         52  Rice - Retail     5     KG     11  2013   
3  National Average         52  Rice - Retail     5     KG     12  2013   
4  National Average         52  Rice - Retail     5     KG      1  2014   

    Price  
0  4.4920  
1  4.5786  
2  4.7865  
3  5.1337  
4  5.5099

In [5]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 7381 entries, 0 to 7380
Data columns (total 8 columns):
 #   Column       Non-Null Count  Dtype  
---  ------       --------------  -----  
 0   Place        7381 non-null   object 
 1   ProductId    7381 non-null   int64  
 2   ProductName  7381 non-null   object 
 3   UmId         7381 non-null   int64  
 4   UmName       7381 non-null   object 
 5   Month        7381 non-null   int64  
 6   Year         7381 non-null   int64  
 7   Price        7381 non-null   float64
dtypes: float64(1), int64(4), object(3)
memory usage: 461.4+ KB


Nhận xét dữ liệu:
    * Bộ dữ liệu có 8 cột và 7381 dòng

    * Place, ProductName, UnName được lưu trữ với kiểu dữ liệu object mang thuộc tính định danh => thước đo nominal

    * ProductId,UmId,Month, Year được lưu trữ với kiểu dữ liệu int64 tuy nhiên dùng để định danh sản phẩm => thước đo nomial
    
    * Price kiểu dữ liệu được lưu trữ với kiểu float dùng để thể hiện giá của sản phẩm, có thể tính toán và đo lường => thước đo ratio


In [6]:
data_price = data.groupby('ProductName')['Price'].mean().reset_index()
data_price  

ProductName       Price
0                               Apples (red) - Retail    2.392193
1                                     Apples - Retail    3.885039
2                                    Bananas - Retail    8.224784
3                              Beans (white) - Retail    9.572830
4                             Bread (common) - Retail    4.463293
5                               Bread (pita) - Retail    2.404545
6                                     Bulgur - Retail    3.516712
7                                    Cabbage - Retail    1.723395
8                                Cauliflower - Retail    3.274868
9                                     Cheese - Retail   25.522875
10                                 Chickpeas - Retail   10.347860
11                            Cocoa (powder) - Retail   43.802339
12                          Coffee (instant) - Retail   43.295493
13                                    Coffee - Retail   42.468075
14                    Cucumbers (greenhouse) - Retail    2.206807
15                                 Cucumbers - Retail    3.411326
16                                 Eggplants - Retail    3.882645
17                                      Eggs - Retail    0.505925
18                               Electricity - Retail    0.513580
19                              Fish (fresh) - Retail   22.698257
20                                Fuel (gas) - Retail   83.976703
21                    Fuel (petrol-gasoline) - Retail    8.592519
22                                    Garlic - Retail   16.610370
23                      Groundnuts (shelled) - Retail   21.832122
24                                   Lentils - Retail    6.651817
25                            Meat (chicken) - Retail    9.683471
26                             Meat (mutton) - Retail   45.557209
27                               Meat (veal) - Retail   41.270904
28                        Milk (pasteurized) - Retail    3.679687
29             Milk (powder, infant formula) - Retail  108.287696
30                               Oil (olive) - Retail   27.274547
31                           Oil (sunflower) - Retail    8.755174
32                                    Onions - Retail    2.141539
33                                   Oranges - Retail    3.124280
34                                     Pasta - Retail    3.496637
35                         Peas (green, dry) - Retail    2.900029
36                                  Potatoes - Retail    2.297231
37                                      Rice - Retail    7.950325
38                                      Salt - Retail    2.048213
39                                   Spinach - Retail    3.240587
40                                     Sugar - Retail    4.482942
41                               Tea (green) - Retail    4.137000
42                              Tea (herbal) - Retail    5.434838
43                                       Tea - Retail   26.224393
44                          Tomatoes (paste) - Retail    8.085128
45                                  Tomatoes - Retail    3.393658
46                        Transport (public) - Retail    2.468031
47  Wage (non-qualified labour, non-agricultural) ...   40.548182
48                                     Water - Retail    5.384253
49                               Wheat flour - Retail    3.253174
50                                    Yogurt - Retail    5.364898
51                                  Zucchini - Retail    3.837423

Đề xuất: Phân tích mô tả